examples/hotrod/cmd/root.go

```diff
@@ -38,6 +38,11 @@ var (
 	fixDBConnDelay         time.Duration
 	fixDBConnDisableMutex  bool
 	fixRouteWorkerPoolSize int
+
+	fixCustomerPort int
+	fixDriverPort int
+	fixFrontendPort int
+	fixRoutePort int
```

Comment:
"fix" prefix was used for variables about because they can be used to "fix" performance problems instead of changing source code as was done in the blog post. For port numbers there's no "fixing".